cpp/Platform.Converters/IConverter[T].h

```diff
@@ -2,7 +2,5 @@
 {
     template <typename ...> class IConverter;
     template <typename T> class IConverter<T> : public IConverter<T, T>
-    {
-    public:
-    };
+    { };
```

Comment:
Mark this whole file as not translated.
Remove content of this file and replace it with comment:
```
// Translation is not required
```